package.json

```diff
@@ -1,7 +1,28 @@
 {
   "name": "@bloom-housing/ui-seeds",
   "version": "0.1.0",
-  "private": true,
+  "homepage": "https://github.com/bloom-housing/ui-seeds",
+  "main": "index.js",
+  "license": "Apache-2.0",
+  "private": false,
+  "publishConfig": {
+    "access": "public",
+    "registry": "https://registry.npmjs.org"
+  },
+  "release": {
+    "branches": [
+      "main",
+      {
+        "name": "alpha",
```

Comment:
Not sure we want to prerelease vs just release but not critical to decide now

Reply:
This was carried over from ui-components. It's setup to do a prerelease if anyone pushes to the "alpha" branch. We used it for testing on ui-components, but I don't see very many times we would need it down the road.